### PR TITLE
Update cloud_cost_management.md

### PR DESCRIPTION
**Pull request by ktaylor19**
### What does this PR do?

Update the Cloud Cost docs tag enrichment section to be accurate with recent changes to the product 

### Motivation

Make sure docs are accurate before dash

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by cnharris10 — APPROVED** [review]

File: content/en/infrastructure/cloud_cost_management.md [hunk 1]
```diff
@@ -130,21 +130,37 @@ You can visualize your ingested data using the following cost types:
...
+The following out-of-the-box tags are also available for filtering and grouping data:
+
+| Tag                          | Description       |
+| ---------------------------- | ----------------- |
+| `aws_product`                | The AWS service being billed.|
+| `aws_product_family`         | The category for the AWS service being billed (for example, Compute or Storage).|
+| `aws_management_account_name`| The AWS management account name associated with the item.|
+| `aws_management_account_id`  | The AWS management account id associated with the item.|
+| `aws_member_account_name`    | The AWS memmber account name associated with the item.|
+| `aws_member_account_id`      | The AWS member account id associated with the item.|
```
Comment (buraizu) [inline content/en/infrastructure/cloud_cost_management.md:146]:
```suggestion
| `aws_member_account_id`      | The AWS member account ID associated with the item.|
```

File: content/en/infrastructure/cloud_cost_management.md [hunk 2]
```diff
@@ -130,21 +130,37 @@ You can visualize your ingested data using the following cost types:
...
+| `aws_cost_type`              | The type of charge covered by this item (for example, Usage, or Tax).|
```
Comment (buraizu) [inline content/en/infrastructure/cloud_cost_management.md:147]:
```suggestion
| `aws_cost_type`              | The type of charge covered by this item (for example, Usage, Tax, and so on).|
```

File: content/en/infrastructure/cloud_cost_management.md [hunk 3]
```diff
@@ -130,21 +130,37 @@ You can visualize your ingested data using the following cost types:
...
+| `aws_pricing_term`           | Whether the usage is Reserved, Spot, or On Demand.|
```
Comment (buraizu) [inline content/en/infrastructure/cloud_cost_management.md:148]:
```suggestion
| `aws_pricing_term`           | Whether the usage is Reserved, Spot, or On-Demand.|
```

File: content/en/infrastructure/cloud_cost_management.md [hunk 4]
```diff
@@ -130,21 +130,37 @@ You can visualize your ingested data using the following cost types:
...
+| `aws_reservation_arn`        | The ARN of the Reserved Instance that the item benefited from.| 
+| `aws_savings_plan_arn`       | The ARN of the Savings Plan the item benefited from.|
+| `aws_usage_type`             | The usage details of the item (for example, BoxUsage:i3.8xlarge).|
+| `aws_operation`              | The operation associated with the item (for example, RunInstances).|
+| `aws_region`                 | The region associated with the item.|
+| `aws_availability_zone`      | The availability zone associated with the item.|
+| `aws_resource_id`            | The resource id associated with the item.|
```
Comment (buraizu) [inline content/en/infrastructure/cloud_cost_management.md:155]:
```suggestion
| `aws_resource_id`            | The resource ID associated with the item.|
```

File: content/en/infrastructure/cloud_cost_management.md [hunk 5]
```diff
@@ -130,21 +130,37 @@ You can visualize your ingested data using the following cost types:
...
+| `aws_instance_type`          | The instance types associated with your items.|
+| `aws_instance_family`        | The instance family associated with your item (for example, Storage optimized).|
+| `is_aws_ec2_compute`         | Whether the usage is related to EC2 compute.|
+| `is_aws_ec2_compute_on_demand`| Whether the usage is on demand.|
```
Comment (buraizu) [inline content/en/infrastructure/cloud_cost_management.md:159]:
```suggestion
| `is_aws_ec2_compute_on_demand`| Whether the usage is on-demand.|
```

File: content/en/infrastructure/cloud_cost_management.md [hunk 6]
```diff
@@ -130,21 +130,37 @@ You can visualize your ingested data using the following cost types:
...
+| `is_aws_ec2_compute_reservation`| Whether the usage associated with a Reserved Instance.|
```
Comment (buraizu) [inline content/en/infrastructure/cloud_cost_management.md:160]:
```suggestion
| `is_aws_ec2_compute_reservation`| Whether the usage is associated with a Reserved Instance.|
```

File: content/en/infrastructure/cloud_cost_management.md [hunk 7]
```diff
@@ -130,21 +130,37 @@ You can visualize your ingested data using the following cost types:
...
+| `is_aws_ec2_capacity_reservation`| Whether the usage associated with a Capacity Reservation.|
```
Comment (buraizu) [inline content/en/infrastructure/cloud_cost_management.md:161]:
```suggestion
| `is_aws_ec2_capacity_reservation`| Whether the usage is associated with a Capacity Reservation.|
```

File: content/en/infrastructure/cloud_cost_management.md [hunk 8]
```diff
@@ -130,21 +130,37 @@ You can visualize your ingested data using the following cost types:
...
+Additionally, Datadog adds out-of-the-box tags to the ingested cost data to help you further break down and allocate your costs. These tags are derived from your [Cost and Usage Report (CUR)][7].
+
+The following out-of-the-box tags are also available for filtering and grouping data:
+
+| Tag                          | Description       |
+| ---------------------------- | ----------------- |
+| `aws_product`                | The AWS service being billed.|
+| `aws_product_family`         | The category for the AWS service being billed (for example, Compute or Storage).|
+| `aws_management_account_name`| The AWS management account name associated with the item.|
+| `aws_management_account_id`  | The AWS management account id associated with the item.|
```
Comment (buraizu) [inline content/en/infrastructure/cloud_cost_management.md:144]:
```suggestion
| `aws_management_account_id`  | The AWS management account ID associated with the item.|
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks, made a few minor suggestions.

**Review by jtappa — CHANGES_REQUESTED** [review]

**Comment by jtappa** [comment]
I'm going to re-open this with a new branch name so that we can get preview builds up.

**Pull request closed without merging**